### PR TITLE
Add sanitizer matrix to GitHub actions examples.

### PR DESCRIPTION
**Pull request by oliverchang**
Also set token permissions to read-all by default to satisfy scorecard
requirements.

### CODE REVIEW

File: docs/running-clusterfuzzlite/github_actions.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ We recommend having separate workflow files for each part of ClusterFuzzLite:
 - `.github/workflows/cflite_pr.yml` (for PR fuzzing)
 - `.github/workflows/cflite_cron.yml` (for tasks done on a cron schedule)
 
-TODO: Host a clean, complete example somewhere.  TODO: multiple sanitizers
+TODO: Host a clean, complete example somewhere.
 TODO: Link to actions docs.
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply docs/running-clusterfuzzlite/github_actions.md:24]: I think I mean link to github's docs on actions so users can read more about them to better understand what they are doing.

File: docs/running-clusterfuzzlite/github_actions.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ on:
   push:
     branches:
       - main  # Use your actual default branch here.
+permissions: read-all
```
[the remark this answers is not in the document]
Comment (jonathanmetzman) [reply docs/running-clusterfuzzlite/github_actions.md:40]: But why are we changing this here? Is this change needed at all?

File: docs/running-clusterfuzzlite/github_actions.md [hunk 3]
```diff
@@ -64,24 +65,32 @@ on:
...
     paths:
       - '**'
+permissions: read-all
 jobs:
   PR:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        sanitizer: [address, undefined, memory]
```
Comment (jonathanmetzman) [inline docs/running-clusterfuzzlite/github_actions.md:75]: I think we should probably have a comment here explaining that this should only contain the fuzzers users want.

**Review by jonathanmetzman — APPROVED** [review]
lgtm

File: docs/running-clusterfuzzlite/github_actions.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (oliverchang) [reply docs/running-clusterfuzzlite/github_actions.md:40]: This is for scorecards to pass on this.

File: docs/running-clusterfuzzlite/github_actions.md (hunk 3 above)
Comment (oliverchang) [reply docs/running-clusterfuzzlite/github_actions.md]: done.

**Pull request merged**